src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -1021,12 +1028,19 @@ def forward(
         )
 
         hidden_states = outputs[0]
-        logits = self.lm_head(hidden_states)
+        if labels is None and not is_torchdynamo_compiling():
+            logger.warning_once(
+                "Starting from v4.44, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)"
```

Comment:
```suggestion
                "Starting from v4.45, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)"
```